ivadomed/inference.py

```diff
@@ -203,6 +204,25 @@ def pred_to_nib(data_lst: List[np.ndarray], z_lst: List[int], fname_ref: str, fn
     return nib_pred
 
 
+def pred_to_png(pred_list: list, target_list: list, subj_path: str, painted: bool = False):
+    """Save the network predictions as PNG files with suffix "_target_pred".
+
+    Args:
+        pred_list (list of np arrays): list of 2D predictions.
+        target_list (list of str): list of target suffixes.
```

Comment:
There is kind of a reason behind this in the current codebase. From what I gathered, the list is called `target_suffix` when it comes directly from the `target_suffix` key in the config file (this is the case in `evaluation.py/evaluate` and `testing.py/run_inference`). And the list is called `target_list` when it comes from the return of `inference.py/segment_volume` (this is the case in `main.py/run_segment_command`).

So for the `pred_to_png` function, I'm dealing with both and I choose `target_list` alongside `pred_list`. This makes more sene to me, knowing that the `segment_volume` function and the `pred_to_png` functions will be used together for the future integration with ADS.

Reply:
Thank you for the thorough explanation. :smile: 

I guess what caught me off guard is that I saw "target_list" and on first glance, I thought it was a list of target images (rather than just the suffixes). Hm... would `target_suffix_list` work, by chance?

If not, this is a nitpicky thing anyway, so NBD to just leave it as-is, too.